src/components/main/MainScreen.js

```diff
@@ -20,6 +19,11 @@ const MainScreen = () => {
   const mainId = focusedItemId || rootId;
   const { data: item, isLoading, isError } = hooks.useItem(mainId);
   const { t } = useTranslation();
+  const [leftContent, setLeftContent] = useState("");
+
+  useEffect(() => {
+    setLeftContent(item.get('name'));
+  }, [])
```

Comment:
Not sure you still need this? 
If yes, you might consider adding `item` in the dependency array, as you cannot wait for isSuccess and on first render the item might be undefined.

Reply:
I have changed the way of doing it. I want to print the item name the first time the item is loaded (because it is the parent item), and then do not update the title. I have set the firstRender state, to know if it is the first time or not. This way we can set the title at the right moment (when the data is available from the server). Anyway, I think we are going to delete the header soon, so this part would be deleted.